airflow/api_connexion/parameters.py

```diff
@@ -41,7 +41,7 @@ def validate_istimezone(value: datetime) -> None:
         raise BadRequest("Invalid datetime format", detail="Naive datetime is disallowed")
 
 
-def format_datetime(value: str) -> datetime:
+def format_datetime(value: str | None) -> datetime | None:
```

Comment:
We needdd to convert the decorators to handle None as default value - see https://github.com/apache/airflow/pull/39055/files#r1566990719